etc/env/example.env

```diff
@@ -2,7 +2,7 @@
 # - - -
 
 # site options
-CANDIG_MODULES=weavescope portainer consul traefik logging monitoring minio drs-server htsget-server chord-metadata datasets rnaget candig-server federation-service #cancogen-dashboard cnv-service wes-server jupyter igv
+CANDIG_MODULES=weavescope portainer consul traefik logging monitoring minio drs-server htsget-server chord-metadata datasets cnv-service rnaget candig-server federation-service cancogen-dashboard #wes-server jupyter igv authentication authorization
```

Comment:
I thought we are not using cnv-service as part of the default stack @ljdursi ?

Reply:
I do not want to remove anything existing in this PR.  That is part of the reason I have DIG-654 open because there is quite a bit in the stack that needs pruning.